spec/requests/authors_spec.rb

```diff
@@ -107,11 +117,11 @@
       end
     end
 
-    context 'when authors does not exist' do
-      let(:invalid_id) { Faker::Number.within(range: 900..1000) }
-      let(:message) { "Couldn't find Author with 'id'=#{invalid_id}" }
+    context 'when authors does not exists' do
```

Comment:
```suggestion
    context 'when authors does not exist' do
```

Nesse caso não precisa do `s` porque está no auxiliary `does` ;)